### PR TITLE
Upgrade to 5Jun2019

### PR DESCRIPTION
**Pull request by njzjz**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{% set build = 7 %}
+{% set build = 0 %}
 {% set name = "lammps" %}
-{% set version = "stable_12Dec2018" %}
+{% set version = "stable_5Jun2019" %}
 {% set sha256 = "643eaf2c677e2590e04df2b5be72acaa19007cb34992efe82d0e5beb79b9c865" %}
```
Comment (njzjz) [inline recipe/meta.yaml]:
```suggestion
{% set sha256 = "999c54ec4bbbe27a81448644ff02b0ed4220747fee7ada06e6fd682478613e2d" %}
```

**Comment by jan-janssen** [comment]
@njzjz the patch should no longer be necessary, it is included now

**Comment by jan-janssen** [comment]
@conda-forge-admin, please rerender

**Comment by jan-janssen** [comment]
@njzjz it finally seems to work - I just wait for the tests to pass before I merge it. Thanks for the suggestion.

File: recipe/build.sh [hunk 2]
```diff
@@ -16,9 +16,9 @@ mkdir build_lib
 cd build_lib
 cmake -D BUILD_LIB=ON -D BUILD_SHARED_LIBS=ON -D BUILD_MPI=OFF -D BUILD_OMP=OFF $args ../cmake
 make # -j${NUM_CPUS}
-cp liblammps.* ../src  # For compatibility with the original make system.
-cd ../python
-python install.py 
+cp liblammps{.dylib,.so} ../src  # For compatibility with the original make system.
```
Comment (njzjz) [inline recipe/build.sh]: use ${SHLIB_EXT}
```suggestion
cp liblammps${SHLIB_EXT} ../src  # For compatibility with the original make system.
```

Comment (jan-janssen) [reply recipe/build.sh]: I agree - that is more for debugging at the moment - I change it once the test pass

**Pull request merged**